### PR TITLE
Support project-directory in Pod Install call.

### PR DESCRIPTION
**Pull request by tirodkar**
fixes https://github.com/google/EarlGrey/issues/287.

@ajsecord, you'll have to wait for the next gem release (probably happening 4-5/10/2016) for getting this change with the gem. You should be able use `rake install` to get this change locally.

### CODE REVIEW

File: gem/lib/earlgrey/configure_earlgrey.rb [hunk 1]
```diff
@@ -20,21 +20,33 @@ def configure_for_earlgrey(*args)
...
+      path = File.join(dir_path, File.basename(project_name, '.*') + ext)
       path ? path : nil
     end
 
+    # Returns the project's directory. If CocoaPods hasn't had it passed in, then the current
+    # directory is chosen.
+    # @return [String] directory path for the Xcode project
+    def dir_path
+      if installer
+        pods_dir_path = installer.config.installation_root
```
Comment (bootstraponline) [inline gem/lib/earlgrey/configure_earlgrey.rb]: what is the value of `installer.config.installation_root` when `--project-directory` is not used?

File: gem/lib/earlgrey/configure_earlgrey.rb [hunk 2]
```diff
@@ -20,21 +20,33 @@ def configure_for_earlgrey(*args)
...
+        if pods_dir_path.to_s.length && Dir.pwd != pods_dir_path
```
Comment (bootstraponline) [inline gem/lib/earlgrey/configure_earlgrey.rb]: Why do we care if it's equal to Dir.pwd?

File: gem/lib/earlgrey/configure_earlgrey.rb [hunk 3]
```diff
@@ -20,21 +20,30 @@ def configure_for_earlgrey(*args)
...
     end
 
+    # Returns the project's directory. If CocoaPods hasn't had it passed in, then the current
+    # directory is chosen.
+    # @return [String] directory path for the Xcode project
+    def dir_path
+      if installer
+        path = installer.config.installation_root
+      end
+      path ? path : Dir.pwd
```
Comment (bootstraponline) [inline gem/lib/earlgrey/configure_earlgrey.rb]: why not just:

```ruby
def dir_path
  installer ? installer.config.installation_root : Dir.pwd
end
```

Comment (tirodkar) [inline gem/lib/earlgrey/configure_earlgrey.rb]: Done.

**Pull request merged**